vault/counters.go

```diff
@@ -230,6 +231,8 @@ func (c *Core) countActiveEntities(ctx context.Context) (*ActiveEntities, error)
 		return nil, err
 	}
 
+	metrics.SetGauge([]string{"identity_store", "num_entities"}, float32(count))
```

Comment:
I'm a little uncomfortable with adding the metric here, because this is the function that the UI uses (via pathInternalCountersEntities) to count the entities. So you'd get a new metric not just on the regular schedule, but also whenever the UI page was accessed.  That's probably not harmful, but it would be better to set the gauge based on the return value, rather than as a side effect of counting.

I would also suggest just "identity" rather than "identity_store".

Reply:
Agreed, I'll update both.